OpenRA.Mods.Common/Traits/Tooltip.cs

```diff
@@ -32,10 +32,17 @@ public class TooltipInfo : ITraitInfo, ITooltipInfo
 		public readonly bool ShowOwnerRow = true;
 
 		[Desc("Sequence of the actor that contains the cameo.")]
-		public readonly string Icon = "icon";
+		public string Icon { get; private set; }
```

Comment:
TraitInfos are supposed to be immutable.  I haven't looked at the code in enough detail to suggest an alternative approach, but please try and find one!